app/src/androidTest/java/ch/epfl/qedit/home/HomeQuizListFragmentTest.java

```diff
@@ -55,10 +50,5 @@ public void testClickOnQuizLaunchesQuizActivity() {
                 .inAdapterView(withId(R.id.home_quiz_list))
                 .atPosition(0)
                 .perform(click());
-
-        intended(
```

Comment:
I don't understand why we remove those lines ? I think that without those lines the test doesn't do much, but I am probably missing something.